tools/export/uvision4.py

```diff
@@ -98,7 +95,7 @@ def generate(self, progen_build=False):
                 project_data['common']['macros'].pop(i)
             i += 1
         project_data['common']['macros'].append('__ASSERT_MSG')
-        project_data['common']['build_dir'] = join(project_data['common']['build_dir'], 'uvision4')
+        project_data['common']['build_dir'] = project_data['common']['build_dir'] + '\\' + 'uvision4'
```

Comment:
This change is part of the release branch (as I noted above). this will get rebased and those changes will be removed. The reason is that for keil it needs to be windows paths. It was join, but then keil reported an error if I tested this in the online compiler)